### PR TITLE
[RFC] Initialize v:completed_item

### PR DESCRIPTION
**Pull request by Shougo**
After https://github.com/neovim/neovim/pull/2563, I noticed the initialization problem in v:completed_item.
It is needed, because v:complete_item must be empty in Vim starting.

Current behavior: `v:completed_item` is empty string until you completed.
https://github.com/Shougo/deoplete.nvim/issues/13

### CODE REVIEW

**Comment by justinmk** [comment]
LGTM

**Comment by fwalch** [comment]
Can you put some reference to why this is needed in the commit message? For example the text in your PR description?

**Comment by Shougo** [comment]
Thanks.

I changed the commit message and add test for v:completed_item.

**Comment by Shougo** [comment]
When is is merged?

**Comment by fmoralesc** [comment]
LGTM. @Shougo, mark it as RDY.

File: src/nvim/main.c [hunk 1]
```diff
@@ -205,6 +205,8 @@ int main(int argc, char **argv)
   set_vim_var_string(VV_PROGPATH, (char_u *)argv[0], -1);
   set_vim_var_string(VV_PROGNAME, path_tail((char_u *)argv[0]), -1);
 
+  set_vim_var_dict(VV_COMPLETED_ITEM, dict_alloc());
```
Comment (justinmk) [inline src/nvim/main.c]: Just plopping this into main() does not seem like the best approach. Isn't there somewhere more appropriate, like `eval_init`? I'm looking around now.

**Comment by justinmk** [comment]
eval_init has this:

```
  set_vim_var_nr(VV_SEARCHFORWARD, 1L);
  set_vim_var_nr(VV_HLSEARCH, 1L);
```

So eval_init() is at least more appropriate than main() for this change.

**Comment by justinmk** [comment]
I'll make the change and then merge.

**Comment by justinmk** [comment]
Merged. Thanks @Shougo

**Comment by Shougo** [comment]
Thank you.

**Pull request closed without merging**